www/views/tab-send.html

```diff
@@ -35,18 +35,35 @@
     <div ng-if="hasFunds && !isSweeping">
       <div class="send-header-wrapper card">
         <div class="item item-heading send-heading"><span translate>Recipient</span></div>
-        <div class="item input search-wrapper" ng-class="{'focus': searchFocus}">
+        <div class="item input search-wrapper">
           <i class="icon icon-svg abs-v-center"><img class="svg" src="img/icon-bitcoin-symbol.svg"></i>
           <div class="item-icon-right">
             <input type="text" class="search-input" id="tab-send-address" placeholder="{{'Search or enter NavCoin address' | translate}}" ng-model="formData.search" ng-change="findContact(formData.search)" ng-model-onblur ng-focus="searchInFocus()" ng-blur="searchBlurred()">
             <i class="icon icon-svg qr abs-v-center separator-left" on-tap="openScanner()"><img src="img/scan-ico.svg"></i>
           </div>
         </div>
         <div class="send-private-toggle">
-          <ion-toggle ng-model="formData.privatePayment" toggle-class="toggle-balanced">
+          <ion-toggle ng-click="togglePrivatePayment()" toggle-class="toggle-balanced">
             <span class="toggle-label" translate>Private Payment</span>
           </ion-toggle>
         </div>
+        <div ng-show="showAddNavTech" class="addNavTechIPContainer">
+          <div ng-hide="navTechAddressSuccess">
+            <p>No NavTech servers found. Please add a NavTech server to make private payments.</p>
+            <p>See <a href="https://navtechservers.com/">navtechservers.com</a> for more details.</p>
```

Comment:
This hyperlink doesn't work on android

Reply:
Any idea how to make links work on Android?